### DISCUSSION TITLE
When BlueThroat will migrate to OCI (Oracle Cloud Infrastructure)

### DISCUSSION THREAD

**Discussion by marcelosv2003** [vishnu-ks/Cloud-migration#181] [Ideas]
Any idea when OCI (Oracle Cloud Infrastructure) will be supported as a target cloud ?

**Comment by iamvishnuks** [comment]
Hi @marcelosv2003,
There's no bluethroat anymore. Now the project known as xmigrate.cloud. 
We don't have any immediate plans to opensource OCI cloud migration feature unless there is a demand from the community.
